### ISSUE TITLE
Validated type section is not present in Chapter 6

### ISSUE THREAD

**Issue by nathanmbrown** [underscoreio/advanced-scala#223]
In the generated book there are references in Chapter 4 for the `Validated` type that will be introduced in Chapter 6, and it is also mentioned in the Chapter 6 summary.
However, the section for `Validated` seems to be omitted from the book.
I can see the source page here however : https://github.com/scalawithcats/scala-with-cats/blob/develop/src/pages/applicatives/validated.md

**Comment by williamdemeo** [comment]
It looks like the section on `Validated` may have been (accidentally?) removed during [this commit](https://github.com/scalawithcats/scala-with-cats/commit/97861c70f15603c79ccaf72d797926caa8e1e631).

**Comment by davegurnell** [comment]
Good spot! We replaced that section with one on `Parallel`. You can accumulate errors directly using `Either` without switching to `Validated` using that type class. You tend to get simpler code compared to switching back and forth between `Either` and `Validated`.

--- original message ---
On November 30, 2022 at 3:05 PM GMT ***@***.*** wrote:



It looks like the section on Validated may have been (accidentally?) removed during this commit.

—

Reply to this email directly, view it on GitHub, or unsubscribe.

You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
--- end of original message ---

**Comment by nathanmbrown** [comment]
Thanks for the clarification. Just to note that the Case Study on Validation seems to assume pre-knowledge of the `Validated` type so it might be good to integrate some of that section into the case study. Or Utilise `Parallel` in the case study instead.

**Comment by noelwelsh** [comment]
Yes, I think we'd eventually like to rewrite all of that content.
